### PR TITLE
fix-review-boxes-review-page

### PR DESCRIPTION
**Pull request by rensw90**
This fixes the incorrect (black) text color for the buy button in the platinum SEO review box.

Also made all the list items bold, instead of just the list items in the gold SEO review box.

### CODE REVIEW

File: css/sass/3-particles/_package.scss [hunk 1]
```diff
@@ -97,13 +96,30 @@
...
 }
 
+.theme-review .bordered.package--platinum {
+
+  li {
+    font-weight: bold;
+  }
+
+  button {
+    background-color: #009288;
```
Comment (moorscode) [inline css/sass/3-particles/_package.scss]: Deze kan weg.

File: css/sass/3-particles/_package.scss [hunk 2]
```diff
@@ -97,13 +96,30 @@
 
 }
 
+.theme-review .bordered.package--platinum {
+
+  li {
+    font-weight: bold;
```
Comment (moorscode) [inline css/sass/3-particles/_package.scss]: Deze mag ook nog weg

File: css/sass/3-particles/_package.scss [hunk 3]
```diff
@@ -97,13 +96,30 @@
...
+
 .theme-review .bordered.package--consulting {
   border-color: $scheme-academy-secondary;
 
   h4, h4 a, a {
     color: $scheme-academy-secondary;
   }
 
+  li {
+    font-weight: bold;
```
Comment (moorscode) [inline css/sass/3-particles/_package.scss]: Deze mag ook nog even weg.

**Comment by rensw90** [comment]
Fixes https://github.com/Yoast/yoast.com/issues/316

**Comment by moorscode** [comment]
CR Done, if the conflict is resolved it can be merged.

**Pull request merged**